### PR TITLE
Make references normative

### LINKED ISSUE #4178: Recovery-31: Normative References
In Recovery-31 there are several references that I judge as normative and not informative ones.

- Section 7.8: "A sender MAY use the pipeACK method described in Section 4.3 of [RFC7661] to determine if the congestion window is sufficiently utilized." A normative reference necessary to follow if one selects to use the optional method.

- Sectiom 7.3.2: "Implementations MAY reduce the congestion window immediately upon entering a recovery period or use other mechanisms, such as Proportional Rate Reduction ([PRR]), to reduce the congestion window more gradually." Another case of a suggested mechanism, which if one is to follow it requires one to read that PPR reference. Secondly, why the alternative reference style.

- Section 8.3: "Markings can be treated as equivalent to loss ([RFC3168]), but other responses can be specified, such as ([RFC8511]) or ([RFC8311])." So RFC3168 is the normative specification for the treat it equal to loss. RFC8511 is the TCP alternative back-off an experimental specification for the alternative. And RFC8311 is the process for performing other experiments with alternative responses. I judge all necessary depending on what one intendeds to do. 

So to my understanding this results in two downrefs to experimental specifications. I think that is acceptable in the context they are used in this document. But to be tried in IETF last call. However, I think I don't want to generally recommend them to be acceptable as downrefs so I have no intentions to promote them for the Downref registry.

### PR DESCRIPTION
**Pull request by janaiyengar**
Closes #4178.

### CODE REVIEW

**Review by ianswett — APPROVED** [review]
I'm not the expert on what should be normative, but these were intentionally marked as informative references since they're all TCP specific RFCs.  The pipeACK reference in particular seems like an odd one.

If this is what the chairs want, then I'm ok with it as an editor, but these choices were intentional.

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -1133,8 +1133,8 @@ limits and so no advantage is gained by doing so.
 
 Endpoints choose the congestion controller that they use. Congestion controllers
 respond to reports of ECN-CE by reducing their rate, but the response may vary.
-Markings can be treated as equivalent to loss ({{?RFC3168}}), but other
-responses can be specified, such as ({{?RFC8511}}) or ({{?RFC8311}}).
+Markings can be treated as equivalent to loss ({{!RFC3168}}), but other
```
Comment (martinduke) [inline draft-ietf-quic-recovery.md:1136]: This section doesn't have normative language, so I don't see a need for these references to be normative.

**Review by martinduke — CHANGES_REQUESTED** [review]
I see what @gloinul is saying, but I also see some wiggle room in RFC 3967, Section 1.1:

   -  When a reference to an example is made, such a reference need not
      be normative.  For example, text such as "an algorithm such as the
      one specified in [RFCxxxx] would be acceptable" indicates an
      informative reference, since that cited algorithm is just one of
      several possible algorithms that could be used.

I am also somewhat concerned that trying to get the downrefs into the registry is more likely to run into trouble (Sec 3)

This procedure should not be used if the proper step is to move the
   document to which the reference is being made into the appropriate
   category.  It is not intended as an easy way out of normal process.
   Rather, the procedure is intended for dealing with specific cases
   where putting particular documents into the required category is
   problematic and unlikely ever to happen.

Anyway, if we decide to proceed the PR is fine modulo my comment below.

**Review by martinthomson — COMMENTED** [review]
I don't think that any of these need to be made normative either.  Each of these is exemplary only.

**Review by ianswett — CHANGES_REQUESTED** [review]
Removing my approval, since I don't really want to make this change and it appears others have similar opinions and letting the conversation on the issue play out.

**Comment by janaiyengar** [comment]
Closing in favor of #4190

**Pull request closed without merging**